db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+max = User.create(username: "maxpower", email: "max@power.com")
```

Comment:
This is a seed file. The ruby commands in here can be run by running `rake db:seed`

`rake db:setup` will also run this file (it runs 3 things, db:create, db:schema:load and db:seed)